spec/index.bs

```diff
@@ -7124,16 +7146,17 @@ GPUQueue includes GPUObjectBase;
                 </div>
         </div>
 
-    : <dfn>copyImageBitmapToTexture(source, destination, copySize)</dfn>
+    : <dfn>copyToTexture(source, destination, copySize)</dfn>
     ::
-        Schedules a copy operation of the contents of an image bitmap into the destination texture.
+        Issues a copy operation of the contents of a platform image/video/canvas
+        into the destination texture.
 
-        <div algorithm=GPUQueue.copyImageBitmapToTexture>
+        <div algorithm=GPUQueue.copyToTexture>
             **Called on:** {{GPUQueue}} this.
 
             **Arguments:**
-            <pre class=argumentdef for="GPUQueue/copyImageBitmapToTexture(source, destination, copySize)">
-                |source|: {{ImageBitmap}} and origin to copy to |destination|.
+            <pre class=argumentdef for="GPUQueue/copyToTexture(source, destination, copySize)">
+                |source|: source image and origin to copy to |destination|.
```

Comment:
One simple question here, for HTMLVideoElement, what's the origin defination?

Reply:
`|source|.{{GPUImageCopyPlatformImage/origin}}`